.travis.yml

```diff
@@ -80,11 +81,14 @@ script:
       if [[ "${WITH_ETCD}" == "true" ]]; then
         export ETCD_DIR="${GOPATH}/bin"
       fi
-  - ./integration/integration_test.sh
-  - cd "$HOME/gopath/src/github.com/google/certificate-transparency-go"
-  - ./trillian/integration/integration_test.sh
-  - cd $HOME/gopath/src/github.com/google/trillian
-  - HAMMER_OPTS="--operations=50" ./integration/maphammer.sh 3
+  - |
+      if [[ -n "${TRILLIAN_SQL_DRIVER}" && "${TRILLIAN_SQL_DRIVER}" != sqlite3 ]]; then
```

Comment:
"sqlite3"?

Reply:
or, just to capture an IRL chat, flip this to check for `mysql` since that what the scripts contained in the block are expecting?

Reply:
Flipped and quoted.